packages/react-ui-components/src/stories/Components/Loader.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import React from 'react';
+import { KvLoader } from '../../components';
+
+export default {
+	title: "Components/Loader",
+	component: KvLoader,
+	argTypes: {
+		state: {
+			control: { type: "select" },
+			options: Object.values(KvLoader)
```

Comment:
double check this, I think you're getting the keys of the component in itself and not the enum

Reply:
Hi @pedrorebelocosta it was fixed yesterday, see the raw of this file https://raw.githubusercontent.com/kelvininc/ui-components/loader-component/packages/react-ui-components/src/stories/Components/Loader.stories.tsx maybe the GitHub is with same bug because doesn't show the correct content here.